src/usr/share/gpgmailer/gpgmailer.py

```diff
@@ -205,3 +204,26 @@ def _build_encrypted_message(self, message_dict, loop_start_time):
                 loop_current_time=loop_start_time)
 
         return message
+
+    def _sendmail(self, message_object, recipients):
+        """
+            Adds From and To headers to the message object, then passes it to sendmail to be
+            queued by the local MTA.
+
+            message_object: A MIMEMultipart message object describing the email to send.
+            recipients:     A list of email addresses to send the email to.
+        """
+        self.logger.info('Sending message via sendmail.')
+        recipients_string = recipients[0]
+        for recipient in recipients:
+            if recipient is not recipients[0]:
+                recipients_string += ', %s' % recipient
+
+        message_object['From'] = self.config['sender']['email']
+        message_object['To'] = recipients_string
+
+        sendmail_process = subprocess.Popen(['sendmail', '-t'], stdin=subprocess.PIPE)
+        sendmail_process.communicate(str(message_object))
+        sendmail_process.stdin.close()
+
+        self.logger.debug('Message sent successfully.')
```

Comment:
I'd probably say it was queued successfully. We no longer know if it has been sent.